test/pubchem/import.spec.ts

```diff
@@ -1,8 +1,12 @@
-import { expect } from "chai";
+import { expect } from 'chai';
+import getNecessaryData from 'pubchem/parseData';
+// import init from 'pubchem/index';
```

Comment:
Hey @nesimtunc I think I may need a little help here.

Line 3 and 9 are commented out since they break things. When importing `pubchem/index` and run `npm run test` I get this error `TypeError [ERR_UNKNOWN_FILE_EXTENSION]: Unknown file extension ".ts" for`
and if I run `npm run test:watch`
I get this error `Error [ERR_REQUIRE_ESM]: Must use import to load ES Module:`

I think this is due to the `limiter` package not being CJS (not sure 🤔). Tried different things to resolve but no luck. I have thought about integrating the source code of `limiter` into the codebase (Their LICENSE is allowing it), modifying tsconfig.json (I don't want to break things for your end), tweaking test setup (not sure how to do that tbh)

Let me know how should I proceed
Thank you

Reply:
Please give me some time to look into this issue. I'll commit to this branch directly. @T410